reflectivity_ui/interfaces/data_handling/instrument.py

```diff
@@ -66,6 +67,66 @@ def get_cross_section_label(ws, entry_name):
         return "%s%s" % (pol_label, ana_label)
 
 
+def mantid_algorithm_exec(algorithm_class, **kwargs):
```

Comment:
this new function does not have a unit test, right?

Reply:
oops, now it does :+1: